pkgs/applications/misc/rofi-emoji/default.nix

```diff
@@ -55,6 +63,8 @@ stdenv.mkDerivation rec {
     wl-clipboard
     xclip
     xsel
+    xdotool
+    wtype
```

Comment:
Do these need to be in the `nativeBuildInputs` too, and not just added to the `PATH` as you did above? (If so, it'd be nice to guard them on the `waylandSupport` and `x11Support` flags, too.)

Reply:
Indeed, those are only runtime dependencies I guess.  
I removed them from `nativeBuildInputs`.